Firestore/Source/Local/FSTLevelDBMigrations.mm

```diff
@@ -102,22 +145,37 @@ + (FSTLevelDBSchemaVersion)schemaVersionWithTransaction:
 }
 
 + (void)runMigrationsWithTransaction:(firebase::firestore::local::LevelDbTransaction *)transaction {
+  [self runMigrationsWithTransaction:transaction upToVersion:kSchemaVersion];
```

Comment:
per my comment below about upToVersion feeling off-by-1, you could change this to kSchemaVersion-1 and it would have no effect (and in fact that's what your test does; since it's passing 2 instead of 3)

Reply:
Discussion below.